src/graphics/program-lib/programs/standard.js

```diff
@@ -1493,14 +1488,9 @@ var standard = {
                 code += "   addReflection();\n";
             }
 
-            if (options.useSpecular && options.useMetalness && options.fresnelModel > 0) code += "   getFresnel();\n";
+            // NB switched to more correct half-angle fresnel
```

Comment:
any reason to have 'NB' on many of the comments?

Reply:
For this one its a note for future users who may wonder why some logic was removed - its true that some NBs can be removed